src/sensai/data_transformation/dft.py

```diff
@@ -491,110 +491,162 @@ class DFTNormalisation(DataFrameTransformer):
+            self.transformer = transformer
+            self.transformerFactory = transformer_factory
+            self.independentColumns = independent_columns
+            self.arrayValued = array_valued
+            self.fit = fit
 
-                * If the relevant features are already normalised, pass ``skip=True``
-                * If the relevant features cannot be normalised (e.g. because they are categorical), pass ``unsupported=True``
-                * If the relevant features shall be normalised, the other parameters apply.
-                  No parameters, i.e. ``RuleTemplate()``, are an option if ...
 
-                    * a default transformer factory is specified in the :class:`DFTNormalisation` instance and its application
-                      is suitable for the relevant set of features.
-                      Otherwise, specify either ``transformerFactory`` or ``transformer``.
-                    * all relevant features are to be normalised in the same way.
-                      Otherwise, specify ``independentColumns=True``.
+    class RuleTemplate(_RuleBase):
+        """
+        A rule template from which a rule can be created by specifying a regex for the columns to which the rule applies.
+        Useful e.g., in FeatureGenerators where a rule for each of the generated columns is to be created.
+
+        Use the parameters as follows:
+
+        * If the relevant features are already normalised, pass ``skip=True``
+        * If the relevant features cannot be normalised (e.g. because they are categorical), pass ``unsupported=True``
+        * If the relevant features shall be normalised, the other parameters apply.
+          No parameters, i.e. ``RuleTemplate()``, are an option if ...
 
-            :param skip: flag indicating whether no transformation shall be performed on all of the columns (because they are already
+            * a default transformer factory is specified in the :class:`DFTNormalisation` instance and its application
+              is suitable for the relevant set of features.
+              Otherwise, specify either ``transformer_factory`` or ``transformer``.
+            * the resulting rule will match only a single column. Otherwise, ``independent_columns``
+              must be specified to True or False.
+
+        """
+
+        # The init is just there for the IDE to properly render docstrings in quick documentation
+        def __init__(self,
+                skip: bool = False,
+                unsupported: bool = False,
+                transformer: Optional[SkLearnTransformerProtocol] = None,
+                transformer_factory: Optional[Callable[[], SkLearnTransformerProtocol]] = None,
+                array_valued: bool = False,
+                fit: bool = True,
+                independent_columns: Optional[bool] = None):
+            """
+            :param skip: if True, no transformation shall be performed on matched columns (e.g., because they are already
```

Comment:
"If True, X" is not good style. It's clumsy and logically incomplete, because, strictly speaking, you are not stating that "If False, not X".
For flags, the style "[flag indicating] whether X" should be preferred.

Reply:
Please restore the original docstrings for the flags.

Reply:
Will do

Reply:
Done, but I kept the second sentence giving an example for when a user might want to specify `unsupported=True`